package.json

```diff
@@ -37,12 +41,15 @@
     "eslint-plugin-import": "^2.25.3",
     "eslint-plugin-lit-a11y": "^2.2.0",
     "eslint-plugin-wc": "^1.3.2",
+    "gh-pages": "^4.0.0",
     "husky": "^8.0.0",
     "madge": "^5.0.1",
     "prettier": "^2.4.1",
-    "sinon": "^14.0.0",
+    "rimraf": "^3.0.2",
+    "sinon": "^12.0.1",
```

Comment:
any reason we change the version of this library to lower?

Reply:
not intentional, changing it back